OWML.Common/IModInputCombination.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace OWML.Common
+{
+	public interface IModInputCombination
+	{
+		float GetLastPressedMoment();
+		float GetPressDuration();
+		bool IsFirst(bool keep = false);
+	}
```

Comment:
Your indents are very long. Do you use Visual Studio or something else? If it's VS, you can auto-format documents with control+K, D, which should make indents smaller. Compare with the rest of the code base.

Reply:
I use tab as indents, probably that's the reason

Reply:
switched indents to spaces

Reply:
I don't see the change